addons/mrp_repair/models/repair.py

```diff
@@ -1,11 +1,27 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, models
+from odoo import api, models, fields, _
 
 
 class Repair(models.Model):
     _inherit = 'repair.order'
 
+    productions_count = fields.Integer(
+        'Count of MO generated',
+        compute='_compute_mto_productions',
+        groups='mrp.group_mrp_user',
+    )
+
+    production_mto_ids = fields.Many2many(
```

Comment:
`production_ids` and I think you don't need a string in this case (it will default to "Productions")